homeassistant/components/reolink/icons.json

```diff
@@ -13,6 +13,12 @@
           "on": "mdi:car"
         }
       },
+      "non-motor_vehicle": {
+        "default": "mdi:motorbike-off",
+        "state": {
+          "on": "mdi:motorbike"
```

Comment:
non-motor vehicle but the suggestion is to use a motorbike as icon? I assume because it has an 'off' and bicycle has not?

Reply:
Yes, bicycle does not have a off, motorbike does and it looks pretty simular.
Motor vehicle has a car as icon, so I think the difference is pretty clear.